### PR TITLE
Fix setup

### PR DESCRIPTION
**Pull request by LDAP**
- Autoinstall dependencies needed for setup.py (numpy, Cython, wheel, setuptools)
- Added requirements.txt for developers

Should I add a specific version for scikit-learn as mentioned in #272?

### CODE REVIEW

**Comment by LDAP** [comment]
[PEP 518](https://www.python.org/dev/peps/pep-0518/)
[Cython Docs](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#basic-setup-py)

**Comment by relf** [comment]
Thanks for the references and you can also add your name in the AUTHORS.md file.

**Comment by relf** [comment]
> Should I add a specific version for scikit-learn as mentioned in #272?

As the errors (#272 ) seems to be related to surrogate models not well adapted to test functions I would prefer not to mention a specific version besides 0.24 works well for some other surrogate models.

File: AUTHORS.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ SMT has been developed thanks to contributions from:
 * Rémy Priem <remy.priem@onera.fr>
 * Steven Berguin <stevenberguin@gatech.edu>
 * Vincent Drouet <vincent.drouet@onera.fr>
+* Lucas Alber <lucasd.alber@gmail.com>
```
Comment (relf) [inline AUTHORS.md:26]: Well so far we have used the lexicographic order  :wink:

**Review by relf — APPROVED** [review]

**Pull request merged**